core/src/main/scala/org/apache/spark/memory/MemoryManager.scala

```diff
@@ -155,4 +164,217 @@ private[spark] abstract class MemoryManager extends Logging {
     _storageMemoryUsed
   }
 
+  // -- The code formerly known as ShuffleMemoryManager --------------------------------------------
```

Comment:
instead of saying what this used to be, maybe describe what this does, i.e. it arbitrates execution memory across threads.